pkgs/os-specific/linux/xp-pen-drivers/deco-01-v2/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, stdenv
+, fetchzip
+, libusb
+, glibc
+, libGL
+, xorg
+, qtx11extras
+, wrapQtAppsHook
+, autoPatchelfHook
+, libX11
+, libXtst
+, libXi
+, libXrandr
+, libXinerama
+}:
+
+let
+  dataDir = "var/lib/xppend1v2";
```

Comment:
On a second look I do think it would be best to just match upstream, instead of renaming this ourselves. That could be confusing to people following upstream documentation, or fixes from other distrobutions.

Reply:
The default is `/usr/lib/pentablet`, though, and NixOS doesn't like anything under `/usr`. I figured that if it was going to be renamed then we may as well rename it to something that has less chance of conflicting with another driver down the line.